websites/I/IFTTT/presence.ts

```diff
@@ -1,277 +1,361 @@
 const presence = new Presence({
     clientId: "665519810054062100"
   }),
-  strings = presence.getStrings({
-    search: "presence.activity.searching",
-    browsing: "presence.activity.browsing",
-    reading: "presence.activity.reading"
-  });
+  strings = getStrings();
 
 presence.on("UpdateData", async () => {
-  const path = window.location.pathname.split("/").slice(1),
+  const host = window.location.hostname,
+    path = window.location.pathname.split("/").slice(1),
     presenceData: PresenceData = {
-      details: "IFTTT",
       largeImageKey: "logo_big"
     };
 
-  switch (window.location.hostname.replace("www.", "")) {
-    //IFTTT URL Shortener (for the Help Center)
-    case "ift.tt":
-      presence.setActivity();
-      return;
+  if (host === "ift.tt") {
```

Comment:
Isn't this just a redirect?

Reply:
This acts as an url shortener for support pages as an example but I don't know if it's also for something else. Therefor this makes sure that the presence shows the icon at least.